src/extension/inspire/src/main/java/org/geoserver/inspire/wfs/WFSExtendedCapabilitiesProvider.java

```diff
@@ -80,19 +81,7 @@ public void encode(Translator tx, WFSInfo wfs, GetCapabilitiesRequest request)
             tx.end("inspire_common:MediaType");
         }
         tx.end("inspire_common:MetadataUrl");
-        tx.start("inspire_common:SupportedLanguages");
-        language = language != null ? language : "eng";
-        tx.start("inspire_common:DefaultLanguage");
-        tx.start("inspire_common:Language");
-        tx.chars(language);
-        tx.end("inspire_common:Language");
-        tx.end("inspire_common:DefaultLanguage");
-        tx.end("inspire_common:SupportedLanguages");
-        tx.start("inspire_common:ResponseLanguage");
-        tx.start("inspire_common:Language");
-        tx.chars(language);
-        tx.end("inspire_common:Language");
-        tx.end("inspire_common:ResponseLanguage");
+        ViewServicesUtils.encodeSupportedLanguages(language,tx);
```

Comment:
Same as WCS, this is a download service. Maybe the class should be name just "ServicesUtils"?

Reply:
fixed see above.